CubeLineMoment.py

```diff
@@ -214,19 +217,24 @@ def cubelinemoment_setup(cube, cuberegion, cutoutcube,
 
 
     # From NGC253 H213COJ32K1 spectral baseline
-    inds = np.arange(noisecube.shape[0])
+    inds = np.arange(noisecubebright.shape[0])
     mask = np.zeros_like(inds, dtype='bool')
     for low,high in noisemapbright_baseline:
-        # Check to see if noisemapbright_baseline is within noisecube channel range
-        if (low <= noisecube.header['NAXIS3']) and (high <= noisecube.header['NAXIS3']):
+        # Check to see if noisemapbright_baseline is within noisecubebright channel range
+        if (low <= noisecubebright.header['NAXIS3']) and (high <= noisecubebright.header['NAXIS3']):
             mask[low:high] = True
         else:
-            raise ValueError("noisemapbright_baseline ({0},{1}) out of range ({2},{3})".format(low,high,0,noisecube.header['NAXIS3']))
+            raise ValueError("noisemapbright_baseline ({0},{1}) out of range ({2},{3})".format(low,high,0,noisecubebright.header['NAXIS3']))
         
     # need to use an unmasked cube
-    noisemapbright = noisecube.with_mask(mask[:,None,None]).std(axis=0)
+    noisemapbright = noisecubebright.with_mask(mask[:,None,None]).std(axis=0)
     print("noisemapbright peak = {0}".format(np.nanmax(noisemapbright)))
 
+    # Create noisemapbright_baseline mask for plotting
+    brightbaseline_mask = np.zeros_like(inds, dtype='bool')
+    for i in range(len(noisemapbright_baseline)):
+        brightbaseline_mask[noisemapbright_baseline[i][0]:noisemapbright_baseline[i][1]] = True
```

Comment:
Could be simplified:
```suggestion
    brightbaseline_mask = np.zeros_like(inds, dtype='bool')
    for lo, hi in noisemapbright_baseline
        brightbaseline_mask[lo:hi] = True
```

Reply:
but double check me on this, I'm not running the code right now

Reply:
Looks good.  Note that I included a few minor changes (like this one) to address diagnostic plot updates, but then decided to do that under a different PR.  I tested your suggestion (edited, as the for-statement needs a colon), and it works fine.  Should I re-commit the branch to include this change?

Reply:
you can just click the 'commit suggestion' button on github, or you can make the change locally & push - either way works